server/models/group.js

```diff
@@ -194,7 +194,10 @@ GroupSchema.statics = {
 
   addAdmin(name, userId) {
     return new Promise((resolve, reject) => {
-      const update = { $push: { admins: userId } };
+      const update = {
+        $push: { admins: userId },
```

Comment:
Since we're not really doing any error checking to see if the admin is already in the group, should we use `$addToSet` here as well?